packages/react-components/theme-designer/src/components/Demo/Demo.tsx

```diff
@@ -90,34 +90,101 @@ const useStyles = makeStyles({
 		3) Note that the spinner was removed since it was causing confusing with the loading state of the page
 */
 
-export const Column1 = () => {
+const AvatarSection = () => {
   const styles = useStyles();
-  const dropdownId = useId('dropdown-default');
   return (
-    <div className={styles.column}>
-      <div className={styles.avatar}>
-        <Persona
-          name="Cameron Evans"
-          secondaryText="Senior Researcher at Contoso"
-          avatar={{ color: 'brand', badge: { status: 'available' } }}
-        />
-      </div>
+    <div className={styles.avatar}>
+      <Persona
+        name="Cameron Evans"
+        secondaryText="Senior Researcher at Contoso"
+        avatar={{ color: 'brand', badge: { status: 'available' } }}
+      />
+    </div>
+  );
+};
+
+const TabSection = () => {
+  const tabValues = ['Home', 'Pages', 'Documents'];
+    return (
       <TabList defaultSelectedValue="tab1">
-        <Tab value="tab1">Home</Tab>
-        <Tab value="tab2">Pages</Tab>
-        <Tab value="tab3">Documents</Tab>
+      {tabValues.map((tab, index) => (
```

Comment:
Putting things in a loop is often the best choice. However, I think this is an instance where keeping it written out is probably the better path. Reason being it's less code and way more readable. Quick and easy reading is often more important in demo code than elegance.